src/test/java/com/trycloud/pages/DashboardPage.java

```diff
@@ -7,35 +7,49 @@
 import java.util.*;
 
 public class DashboardPage extends BasePage {
-    @FindBy(xpath = "//ul[@id='appmenu']/li")
-    public List<WebElement> modules;
+    //corner user part
     @FindBy(id = "settings")
     public WebElement userCircle;
+    @FindBy(className = "user-status-menu-item__toggle")
+    public WebElement StatusOptions;
     @FindBy(css = "div.user-status-menu-item>span")
     public WebElement userName;
-    @FindBy(xpath = "//a[.='Customize']")
-    public WebElement customizeButton;
-    @FindBy(css = "li>input#status-checkbox-status")
+
+    //widgets
+    @FindBy(className = "icon-user-status-online")
     public WebElement Status;
-    @FindBy(css = "li>input#status-checkbox-weather")
+    @FindBy(className = "icon-weather-status")
     public WebElement Weather;
-    @FindBy(css = "li>input#panel-checkbox-calendar")
+    @FindBy(className = "icon-calendar-dark")
     public WebElement UpcomingEvents;
-    @FindBy(css = "li>input#panel-checkbox-deck")
+    @FindBy(className = "icon-deck")
     public WebElement UpcomingCards;
-    @FindBy(css = "li>input#panel-checkbox-recommendations")
+    @FindBy(className = "icon-files-dark")
     public WebElement RecommendedFiles;
-    @FindBy(css = "li>input#panel-checkbox-spreed")
+    @FindBy(className = "icon-talk")
     public WebElement TalkMentions;
-    @FindBy(css = "li>input#panel-checkbox-user_status")
+    @FindBy(className = "icon-user-status")
     public WebElement RecentStatus;
 
 
+
+    //other elements
+    @FindBy(xpath = "(//button)[3]")
+    public WebElement closeButton;
+    @FindBy(xpath = "//a[.='Customize']")
+    public WebElement customizeButton;
+
+
     //return webelement of modules
     public WebElement modules(String module) {
```

Comment:
This is basically the same method present in the base page

Reply:
Nope the base one clicks mine just used to create the weblelemnt